src/test/miner_tests.cpp

```diff
@@ -214,6 +214,9 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
     // Simple block creation, nothing special yet:
     BOOST_CHECK(pblocktemplate = AssemblerForTest(chainparams).CreateNewBlock(scriptPubKey));
 
+    // Dogecoin: Disable tests which spend rewards, until we have 240+ blocks in the mining tests
+    /*
+
     // We can't make transactions until we have inputs
     // Therefore, load 110 blocks :)
     static_assert(sizeof(blockinfo) / sizeof(*blockinfo) == 110, "Should have 110 blocks to import");
```

Comment:
Not as-is, because this is the size of the precalculated blocks at the top of the test. We'd have to mine 250 blocks at 60 second intervals, to fix this test, which I think we agreed would be too much refactoring in multiple PRs.

Reply:
Okay. I guess that makes this a `TODO` then?

Reply:
Added a TODO

Reply:
Also created https://github.com/dogecoin/dogecoin/issues/1832 to track